common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -105,6 +109,27 @@
                     "Unless explicitly requested, heap buffer will always be preferred to avoid potential system " +
                     "unstability.");
         }
+
+        long maxDirectMemory = SystemPropertyUtil.getLong("io.netty.maxDirectMemory", -1);
```

Comment:
add a comment similar to https://github.com/netty/netty/pull/5314#issuecomment-223110458 describing the behavior of this property?